keyboards/labyrinth75/keymaps/default/keymap.c

```diff
@@ -0,0 +1,55 @@
+        KC_TAB,   KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,  KC_PGUP, \
+        KC_CAPS,    KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT,    KC_ENT,   KC_PGDN, \
+        KC_LSHIFT,     KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH,   KC_RSHIFT,   KC_UP, KC_END, \
+        KC_LCTL,  KC_LWIN,  KC_LALT,               KC_SPACE,              KC_RALT,  KC_RCTL,    KC_LEFT, KC_DOWN, KC_RIGHT  \
+    ),
+    [1] = LAYOUT(
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,  _______, \
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,  _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______ \
+    
+    ),
+    [2] = LAYOUT(
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,  _______, \
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,  _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______ \
+    
+    ),
+    [3] = LAYOUT(
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,  _______, \
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,  _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______ \
+    
+    ),
```

Comment:
```suggestion
```

These empty, inaccessible layers can be omitted from the default keymap.

You could also omit the trailing backslash from the first layer to pretty it up, if you wish.